README.md

```diff
@@ -1,71 +1,68 @@
 # k0smotron
 
-The Kubernetes control plane manager. Deploy and run Kubernetes control planes powered by [k0s](https://k0sproject.io) on any existing cluster.
+An open source control plane manager for unified cluster management.
+
+k0smotron allows you to unify your Kubernetes cluster management for an efficient use of resources. It’s designed for k0s.
 
 ## Features
 
-### Kubernetes-in-Kubernetes
+### Control Plane-as-a-Service
 
-k0smotron allows you to easily create and manage the clusters in an existing Kubernetes cluster. This allows unparalled scalability and flexibility when you have to work with many clusters. It allows truly homogenous setup for all control planes and thus eases the maintenance burden.
+k0smotron streamlines k0s control plane creation and management within your management cluster, reducing traditional operational overhead. k0smotron encapsulates the control plane service as a pods (and other Kubernetes constructs) and provides an intuitive approach to cluster lifecycle management through ClusterAPI integration.
```

Comment:
> reducing traditional operational overhead

Overhead of what? Would be nice to explain a bit more what we mean with this.

Reply:
Traditional operational overhead, like infrastructure management, installation and configuration, upgrades and patches, security monitoring and troubleshooting, backup and dr or high availability. Unfortunately, listing all of these will exceed the aligned lengths of the three listed features. Which of these operational tasks do you find most useful to list?

Reply:
I've added 4 examples.